5-flexbox/style.css

```diff
@@ -0,0 +1,54 @@
+body {
+    margin: 0;
+    padding: 0;
+    width: 100%;
+    height: 100%;
+    font-family: 'Balsamiq Sans';
+    background-color: #E8E8E8;
+}
+
+h1 {
+    text-align: center;
+    font-size: 40px;
+    margin-top: 50px;
+    font-weight: 100;
+    font-style: italic;
+}
+
+main {
+    width: 100%;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    margin: auto;
+}
+
+article {
+    width: 390px;
+    border-radius: 8px;
+    box-shadow: 0 2px 2px rgba(0, 0, 0, 0.2);
+    overflow: hidden;
+    margin: 20px;
+    padding: 10px;
+    text-align: center;
+    transition: all 0.25s;
+    background-color: white;
+}
+
+article:hover {
+    transform: translate(-15px);
+    box-shadow: 0 12px 16px rgba(0, 0, 0, 0.2);
+}
+
+img {
+    width: 210px;
```

Comment:
if the images have not 1:1 aspect ratio, they will look stretched or shrinked. Keep this in mind for future image dimensions handling.